pennylane_ionq/api_client.py

```diff
@@ -262,9 +267,9 @@ def get(self, resource_id=None):
             raise MethodNotSupportedException("GET method on this resource is not supported")
 
         if resource_id is not None:
-            response = self.client.get(self.join_path(str(resource_id)))
+            response = self.client.get(self.join_path(str(resource_id)), params=params)
         else:
-            response = self.client.get(self.resource.PATH)
+            response = self.client.get(self.resource.PATH, params=params)
```

Comment:
codecov is also complaining about this line, but I'm not entirely sure why - it was already here before. if you can test it, that would be sweet. otherwise I'm sure we can figure out a way around it for this PR

Reply:
coverage added